File: fix.py
import re

filename = r"c:\Users\samsung\Downloads\test3\index.html"
with open(filename, "r", encoding="utf-8") as f:
    content = f.read()

# Replace <span class="tts-sent"> with <span class="sentence" data-sentence="N">
i = 1
def replacer(match):
    global i
    res = f'<span class="sentence" data-sentence="{i}">'
    i += 1
    return res

new_content = re.sub(r'<span class="tts-sent">', replacer, content)

with open(filename, "w", encoding="utf-8") as f:
    f.write(new_content)

print(f"Replaced {i-1} occurrences.")

File: fix.js
const fs = require("fs");
const filepath = "c:\\Users\\samsung\\Downloads\\test3\\index.html";
let content = fs.readFileSync(filepath, "utf8");

let i = 1;
content = content.replace(/<span class="tts-sent">/g, function () {
  return `<span class="sentence" data-sentence="${i++}">`;
});

fs.writeFileSync(filepath, content, "utf8");
console.log(`Replaced ${i - 1} occurrences.`);
